dependencies/setupFmi4c.bat

```diff
@@ -23,8 +23,8 @@ cmake -Wno-dev -G %HOPSAN_BUILD_CMAKE_GENERATOR% -DCMAKE_BUILD_TYPE=Debug -DCMAK
 cmake --build . --parallel 8
 cmake --build . --target install
 
-xcopy /S /E /Y %codedir%\include\ %installdir%\include\
-xcopy /S /E /Y %codedir%\3rdparty\fmi\ %installdir%\3rdparty\fmi\
+xcopy /S /E /Y %codedir%\include %installdir%\include\
```

Comment:
CMake inside fmi4c should handle this in the installation step, I can try to fix that